include/grpcpp/support/channel_arguments.h

```diff
@@ -70,6 +70,9 @@ class ChannelArguments {
   /// the resolver.
   void SetGrpclbFallbackTimeout(int fallback_timeout);
 
+  /// Set the channel args to be used for the channel to grpclb server.
+  void SetGrpclbChannelArgs(const ChannelArguments& grpclb_channel_args);
```

Comment:
Let's not add a first-class API for this.  That would require a gRFC, and I don't think it's something we want anyway, since this is an edge case, not really a first-class knob that we're providing.

I think it's fine for users `SetPointer()` or `SetPointerWithVtable()` for this arg.

Reply:
I modified SetPointer to specially check for this channel arg. I think asking the user to use SetPointerWithVtable might introduce some bugs if they dont pass the correct Vtable